Tests/BackendTest/ParserTest.cs

```diff
@@ -0,0 +1,80 @@
+using FluentAssertions;
+using FRITeam.Swapify.APIWrapper;
+using FRITeam.Swapify.APIWrapper.Enums;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace BackendTest
+{
+
+    public class ParserTest
+    {
+
+        [Fact]
+        public void ApiParserTestThrowException()
+        {
+            ResponseParser parser = new ResponseParser();
+            string input = "{\"report\":\"Nena\u0161li sa \\u017eiadne bloky pre rozvrh.\",\"ScheduleContent\":[]}";
+
+
+            Action act = () => parser.ParseResponse(input);
+
+            Assert.Throws<ArgumentException>(act);
+            
+        }
+
+        [Fact]
+        public void ApiParserTest()
```

Comment:
Meaningful name could be: ParseResponce_ParseEmptyAndCorrectBlock

Reply:
ok